templates/enterprise/index.html

```diff
@@ -0,0 +1,221 @@
+      <p class="p-heading--four">Driving digitalisation for mobile workforces</p>
+    </div>
+    <div class="col-6 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/4d799496-Anbox_MobAppManagement_Apps.svg",
+        alt="",
+        width="152",
+        height="195",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="p-strip u-no-padding--bottom">
+    <div class="row">
+      <div class="col-6">
+        <h4>Mobile apps are central to businesses undergoing digital transformation. However, security and data protection is costly with classical mobile app management solutions. Anbox Cloud enables compliant and low-Capex mobile application management from the cloud.</h4>
+      </div>
+      <div class="col-6">
+        <p>Enterprises around the world are undergoing rapid digital transformation. For enterprises with a mobile workforce, digital transformation drives the increasing use of corporate productivity apps on smartphones and tablets.</p>
+        <p>However, with corporate-provided mobile devices, creating strict boundaries between private and corporate data is a challenge. Employees may use corporate devices to private ends, thereby exposing sensitive data.</p>
+        <p>Enterprise IT leaders need to put a mobile strategy in place that fosters rapid digital transformation while protecting corporate data and employee privacy. Mobile application management (MAM) offers a secure way to distribute, secure, monitor and update mobile apps for the digital workplace.</p>
+        <a>Learn about MAM with Anbox Cloud&nbsp;&rsaquo;</a>
+      </div>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="p-strip u-no-padding--top u-hide--small">
+    <div class="row u-align--center">
+      <div class="col-4">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/37d10b82-Anbox_MobAppManagement_Settings.svg",
+          alt="",
+          width="171",
+          height="190",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="col-4">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/7b961704-Anbox_MobAppManagement_Functions.svg",
+          alt="",
+          width="201",
+          height="190",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="col-4">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/e76a9d96-Anbox_MobAppManagement_QR_Code.svg",
+          alt="",
+          width="148",
+          height="190",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <h2>Secure mobile application management</h2>
```

Comment:
The child of a `row` should always be a `col` (you can just do `col-12` or use the `u-fixed-width` class instead)